13.md

```diff
@@ -110,7 +110,7 @@ function countLeadingZeroes(hex) {
 Querying relays for PoW notes
 -----------------------------
 
-Since relays allow searching on prefixes, you can use this as a way to filter notes of a certain difficulty:
+If relays allow searching on prefixes, you can use this as a way to filter notes of a certain difficulty:
```

Comment:
If we are to get rid of prefix search completely I would suggest to remove this line.
Instead, to filter notes by difficulty relays may offer NIP-50 style i.e `{search: "min_difficulty:10"}`

Reply:
Good idea. I tried to do POW-based prefix matching on ZBD Social for filtering high quality replies, but it didn't make any difference since barely anyone is publishing POW notes and was probably just abusing relays for no reason. This would allow relays that really want to be hubs of POW content like powrelay.xyz to implement proper support for it instead of generic prefix matching.